package.json

```diff
@@ -37,6 +37,7 @@
     "JSONStream": "^1.3.5",
     "cookie": "^0.5.0",
     "dotenv": "^16.3.1",
+    "fastq": "^1.15.0",
```

Comment:
You can test you latest version of `@logux/core` by using `"@logux/core": " VladBrokcore/#feat/54"` in `package.json`